pkg/apis/azuredisk/v1beta2/types.go

```diff
@@ -521,3 +521,90 @@ type ListSnapshotsResult struct {
 	// +optional
 	NextToken string `json:"next_token,omitempty"`
 }
+
+type AzDiskDriverConfiguration struct {
```

Comment:
We should in an unnamed `metav1.TypeMeta` field as the first field for consistency with other Kubernetes objects (and to capture the `kind` and `apiVersion` fields in the YAML).